app/models/ems_refresh/save_inventory.rb

```diff
@@ -11,7 +11,7 @@ def save_ems_inventory(ems, hashes, target = nil)
     when ManageIQ::Providers::ConfigurationManager          then save_configuration_manager_inventory(ems, hashes, target)
     when ManageIQ::Providers::ContainerManager              then save_ems_container_inventory(ems, hashes, target)
     when ManageIQ::Providers::NetworkManager                then save_ems_network_inventory(ems, hashes, target)
-    when ManageIQ::Providers::StorageManager::CinderManager then save_ems_cinder_inventory(ems, hashes, target)
+    when ManageIQ::Providers::BlockStorageManager           then save_ems_block_storage_inventory(ems, hashes, target)
```

Comment:
Given some storage managers will be able to export multiple types of storage, we can't rely on a base-class here.
Instead, this line should be something like:
```ruby
  when ManageIQ::Providers::StorageManager      then save_ems_storage_inventory(ems, hashes, target)
```
Then, the `save_ems_storage_inventory` method would look something like:
```ruby
def save_ems_storage_inventory(ems, hashes, target)
  save_ems_block_storage_inventory(ems, hashes, target)  if ems.supports?(:block_storage)
  save_ems_object_storage_inventory(ems, hashes, target) if ems.supports?(:object_storage)
  # etc.
end
```